panel/chat/message.py

```diff
@@ -45,14 +47,19 @@
 Avatar = Union[str, BytesIO, bytes, ImageBase]
 AvatarDict = Dict[str, Avatar]
 
+DIST_PATH = get_dist_path()
```

Comment:
The dist path cannot be resolved on import, it must be resolved dynamically when rendering.